kolibri/core/assets/src/views/ExamReport/index.vue

```diff
@@ -66,7 +66,10 @@
       />
     </template>
 
-    <template v-if="!windowIsSmall && !loading && currentTry.attemptlogs.length" #aside>
+    <template
+      v-if="!windowIsSmall && !loading && currentTry && currentTry.attemptlogs.length"
```

Comment:
Why is it better that we also include a check for `currentTry`?

Reply:
Without it, when `currentTry` wasn't defined, attempting to read `currentTry.attemptLogs` would produce an error for trying to read a property from `undefined`.